### PR TITLE
Classification Example Updates

### PR DESCRIPTION
**Pull request by marko-polo-cheno**
### What change does this PR introduce and why?

Adding QS to READMEs of examples. Fix bug.

### CODE REVIEW

File: examples/dataset/README.md [hunk 1]
```diff
@@ -2,16 +2,18 @@
 
 This directory contains example Kolena Datasets integrations for different machine learning workflows.
 
-- [Binary Classification](./classification): open-source classification models tested using the
-  [Dogs vs. Cats](https://www.kaggle.com/c/dogs-vs-cats) dataset
+- [Classification](./classification): open-source classification models tested using the
```
Comment (ly-kolena) [inline examples/dataset/README.md:5]: What's the ordering logic?

**Review by munkyshi — APPROVED** [review]

File: examples/dataset/README.md (hunk 1 above)
Comment (munkyshi) [reply examples/dataset/README.md:5]: Looks like alphabetical by top level "ML workflow"

Comment (ly-kolena) [reply examples/dataset/README.md:5]: Ack

Comment (marko-polo-cheno) [reply examples/dataset/README.md:5]: Yes, to match Github's folder ordering

**Pull request merged**